tests/snuba/api/endpoints/test_project_group_index.py

```diff
@@ -1539,30 +1538,6 @@ def test_bulk_delete_performance_issues(self) -> None:
             assert response.status_code == 204
             self.assert_groups_are_gone(groups)
 
-    @patch("sentry.api.helpers.group_index.delete.may_schedule_task_to_delete_hashes_from_seer")
-    def test_do_not_mark_as_pending_deletion_if_seer_fails(self, mock_seer_delete: Mock) -> None:
```

Comment:
I could have changed the behaviour of this test but it seemed trivial to keep a test just to prove that failing to communicate with Seer will keep going (try/except block in `delete_group_hashes`.